src/components/pages/HomePage.js

```diff
@@ -0,0 +1,311 @@
+      />
+                                    </Form.Group>
+                                    <Form.Group widths='equal'>
+                                        <Form.Input
+      fluid
+      label='Email'
+      placeholder='someone@example.com'
+      onChange={setEmail}
+      />
+                                    </Form.Group>
+                                    <Form.Group widths='equal'>
+                                        <Form.Input
+      fluid
+      label='Password'
+      type='password'
+      placeholder='Password'
+      onChange={setPassword}
+      />
+                                        <Form.Input
+      fluid
+      label='Repeat Password'
+      type='password'
+      placeholder='Repeat Password'
+      onChange={setPasswordRepeat}
+      />
+                                    </Form.Group>
+                                    <Form.Group widths='equal'>
+                                        <Form.Input
+      fluid
+      label='Address'
+      placeholder='Address'
+      onChange={setAddress}
+      />
+                                        <Form.Input
+      fluid
+      label='City'
+      placeholder='City'
+      onChange={setCity}
+      />
+                                    </Form.Group>
+                                    <Form.Group widths='equal'>
+                                        <Form.Dropdown
+      fluid
+      label='State'
+      placeholder='State'
+      search
+      selection
+      options={statekeys}
+      onChange={setAddressState}
+      />
+                                        <Form.Input
+      fluid
+      label='Zipcode'
+      placeholder='Zipcode'
+      onChange={setZipcode}
+      />
+                                    </Form.Group>
+                                    <Form.Group style={Styles.signUpCheckbox}>
+                                        <Form.Checkbox label='I agree to the Terms and Conditions'/>
+                                    </Form.Group>
+                                    <Form.Group>
+                                        <Form.Button
+      style={Styles.modalButton}
+      size='large'
+      onClick={closeSignupModal}
+      >
+                                            Submit
+                                        </Form.Button>
+                                    </Form.Group>
+                                </Form>
+                            </Modal.Description>
+                        </Modal>
+                    </Menu.Menu>
+                </Menu>
+            </div>
+            <Grid style={Styles.header} centered padded relaxed>
+            <Grid.Row>
+            </Grid.Row>
+             <Grid.Row>
+             <Link to='/requestPipeline'>
+             <Button size='large'>Make a Request</Button>
+             </Link>
+            </Grid.Row>
+
+             <Grid.Row>
+            </Grid.Row>
+
+            </Grid>
+            </div>
+    );
+  }
+}
+;
+
+const mapDispatchToProps = dispatch => {
```

Comment:
Something that I've been using on my project at SAI is to wrap a dumb component in a connected component. The connected component would take care of the `mapDispatchToProps` and `mapStateToProps`. That let's the dumb component to only worry about displaying something to the user. The connected component can pass it all of the props it gets to handle Redux stuff, and in our tests, we can mock or fake those.

Reply:
The tutorial I sent him actually brings that up in the first video. It shows how to to exactly how to do that for reference.

Reply:
Could you share an example i dont think I could find that in the tutorial anywhere @jd1388